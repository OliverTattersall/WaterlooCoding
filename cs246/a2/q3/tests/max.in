a 0 b 1 1 4 4
a 1 b 0 0 2 3
a 2 b 0 1 4 1
a 3 b 1 9 1 1
a 4 b 0 0 1 1
pc0
pc1
pc2
pc3
pc4
