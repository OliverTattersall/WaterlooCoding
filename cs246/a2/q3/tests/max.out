Dimensions: 5x5
Number of rectangles: 1
	[Blue (1,1) 4x4]
Dimensions: 2x3
Number of rectangles: 1
	[Blue (0,0) 2x3]
Dimensions: 5x1
Number of rectangles: 1
	[Blue (0,1) 4x1]
Dimensions: 10x2
Number of rectangles: 1
	[Blue (1,9) 1x1]
Dimensions: 1x1
Number of rectangles: 1
	[Blue (0,0) 1x1]
